sites/default/sqlconf.php

```diff
@@ -1,26 +1,29 @@
 <?php
 //  LibreEHR
-//  MySQL default Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
+//  MySQL Config
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
+$pass	= 'Henrietta94';
 $dbase	= 'libreehr';
 
+global $disable_utf8_flag;
+$disable_utf8_flag = false;
```

Comment:
How did this even get submitted.... @Ngai-E  i thought the gitignore handles this.

Reply:
Please remove this from this commit